javascript/README.md

```diff
@@ -27,44 +27,14 @@ npm install ../relative/path/to/sensr_sdk/javascript/sensr_js_sdk
 You should now have a main file (called `index.js` by default) that you can run using `npm start` or `node index.js`. This file is the entry-point into your project.
```

Comment:
Do we need `## Setting up your own project` section?

Reply:
it is still applicable but its not really important @thor-sr what is your opinion on this section?

Reply:
If you think the users can do this without our help, I'm fine with removing it